packages/e2e-tests/specs/block-deletion.test.js

```diff
@@ -22,10 +23,29 @@ const addThreeParagraphsToNewPost = async () => {
 	await page.keyboard.press( 'Enter' );
 };
 
-const clickOnBlockSettingsMenuItem = async ( buttonLabel ) => {
+/**
+ * Due to an issue with the Popover component not being scrollable
+ * under certain conditions, Pupeteer cannot "see" the "Remove Block"
+ * button. This is a workaround until that issue is resolved.
+ *
+ * see: https://github.com/WordPress/gutenberg/pull/14908#discussion_r284725956
+ */
+const clickOnBlockSettingsMenuRemoveBlockButton = async () => {
+	let isRemoveButton = false;
+
 	await clickBlockToolbarButton( 'More options' );
-	const itemButton = ( await page.$x( `//*[contains(@class, "block-editor-block-settings-menu__popover")]//button[contains(text(), '${ buttonLabel }')]` ) )[ 0 ];
-	await itemButton.click();
+
+	while ( false === isRemoveButton ) {
```

Comment:
This could be written as a `while ( true ) {` and a `break` when the active element is the intended focus target, rather than assigning into the scoped variable. Doing so I think makes more obvious the problem I first noticed, which is that when things don't go according to plan, this will get stuck in an infinite loop of tabbing. The test timeouts are such that it will eventually be forced to stop (I expect), but I'm wondering if there may be some option to reinstate some sanity here (either a different way to find the next tabbable target by condition, or some bailout on e.g. when having completed a full cycle of tabs).

It's not a blocker, but I foresee this being an issue in the future.